ui/src/app/kvm/components/KVMHeaderForms/AddKVM/AddLxd/AuthenticationForm/AuthenticationFormFields/AuthenticationFormFields.tsx

```diff
@@ -39,17 +42,16 @@ export const AuthenticationForm = ({
           type="radio"
         />
         <p>Run the command below in the LXD CLI:</p>
-        <CodeSnippet
-          blocks={[
-            { code: "lxd config trust add - << EOF" },
-            {
-              // TODO: Generate certificate in the UI and display here
-              // https://github.com/canonical-web-and-design/app-squad/issues/257
-              code: "certificate data",
-            },
-            { code: "EOF" },
-          ]}
-        />
+        <div className="certificate-wrapper">
+          <CodeSnippet
+            blocks={[
+              { code: "lxd config trust add - << EOF" },
+              { code: generatedCertificate?.certificate || "" },
```

Comment:
Is this going to pop-in when it loads? Do we need to display something different if it doesn't exist?

Reply:
It shouldn't. This step should only show if `generatedCertificate` exists, but the selector doesn't know that.